src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));
```

Comment:
see common mistakes

Reply:
Is the problem here in the empty line 6 or in the return statement of the getRandomBall method?

Reply:
Color Supplier should be instantiated once - think about making it class level field. 

The same for Random. 

100 is a magic number, use constant to avoid direct using of magic numbers.

Reply:
Thank you. I think I got it